cmd/list/cluster/cmd.go

```diff
@@ -18,6 +18,7 @@ package cluster
 
 import (
 	"fmt"
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
```

Comment:
NP: to sort the ordering on these imports. These imports are already wrong in this file, but we try to keep them in groups, from std, external and then internal pkgs. :pray:

Reply:
We should adopt the same for backend to auto format this. Will look into that later

Reply:
@gdbranco I created us a JIRA for this a while back: https://issues.redhat.com/browse/OCM-3618